consensus/misc/eip1559.go

```diff
@@ -28,6 +28,12 @@ import (
 // We would only go above this if L1 base fee hits 2931 Gwei.
 const MaximumL2BaseFee = 10000000000
 
+var (
+	BaseFeePrecision       = new(big.Int).SetUint64(1e18)
+	DefaultBaseFeeScalar   = new(big.Int).SetUint64(34000000000000)
+	DefaultBaseFeeOverhead = new(big.Int).SetUint64(15680000)
```

Comment:
Should we add some comments about the meaning of this field? e.g., `DefaultBaseFeeOverhead covers sequencing and proving fees for transactions`.

Reply:
Added.